### ISSUE TITLE
Placeholder in Requirement

### ISSUE THREAD

**Issue by dellise26** [DRE2N/DungeonsXL#965]
Hello, i'm using this plugin for a long time and find it exclusive and impressive, so anyway thx.
So here i'am because i want to use requirement on dungeon with placeholder api, it would certainly make everyone happy, and i was thinking of adding a custom message for each requirement with PAPI cause it would be ugly to see "You cannot enter this dungeon cause rpg.level is not 40" or something like that.
Thx again and hope something will be done about this :)

**Comment by Sataniel98** [comment]
So basically need a way to replace the requirement message. That makes sense to me, but what does this have to do with PAPI?

**Comment by dellise26** [comment]
Instead of making a requirement for everything, why not be able to compare a placeholder with a value.
For example i'm using MMOCore and i want to check the level of the player, to be done, a placeholder exist, so making something like "%mmocore_level% >= 40", and it would allow player superior or equals to 40 which can enter the dungeon.
Is it understandable ?
https://git.lumine.io/mythiccraft/mmocore/-/wikis/Placeholders

**Comment by Sataniel98** [comment]
That means we'd need a parser for logical expressions. That would be a considerably bigger thing. Might be an option for the future, but nothing for the current milestone.

**Comment by dellise26** [comment]
Yes indeed, it could be just an equals, but i don't think allowing just one value is .. mhhh not good, but not a problem to wait, i just find it quite useful to use placeholder
